week-1-practise/level-1/4-logic-error-1.js

```diff
@@ -1,7 +1,7 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
```

Comment:
Great. Quick one to check, ‘word.trim’ returns a different type of JS object. Check it and see what it returns (you can use typeof to help)

Reply:
I've checked, word.trim return string as I expected. so I think it won't change the type of object.